WordPress/src/main/res/layout/reader_related_post.xml

```diff
@@ -1,51 +1,109 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    individual related post shown in ReaderRelatedPostsView
+ -->
+
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?android:selectableItemBackground"
-    android:padding="@dimen/reader_related_post_margin">
+    android:layout_marginBottom="@dimen/margin_extra_large"
+    android:background="?android:selectableItemBackground">
+
+    <org.wordpress.android.widgets.WPNetworkImageView
+        android:id="@+id/image_featured"
+        android:layout_width="@dimen/reader_related_post_image_width"
+        android:layout_height="match_parent"
+        android:layout_marginRight="@dimen/margin_large"
+        android:scaleType="fitXY"
+        android:visibility="gone"
+        tools:src="@drawable/box_with_pages_top"
+        tools:visibility="visible" />
 
-    <LinearLayout
+    <RelativeLayout
+        android:id="@+id/layout_related_post_site_header"
```

Comment:
I wrestled with this one. I agree it would be more consistent to show the site when the header is clicked, but at the same time it's very easy to tap the header when you intended to tap the post (something that actually happens to me already in the list view, but that's another story!). So in the end I decided not to make the header tappable.